sources/documents/RTB/Verbali/Interni/VerbaleInterno_2024-05-02/capitolo-riunione.tex

```diff
@@ -0,0 +1,73 @@
+\section{Riunione}
+\subsection{Ordine del giorno}
+\begin{itemize}
+	\item Analisi delle attività svolte durante la settimana dai membri del gruppo;
+	\item Discussione dei punti che verranno trattati al diario di bordo del 3 maggio 2024;
+	\item Invio di una mail per l'organizzazione di un incontro con La Proponente per aggiornarLa sullo stato del progetto.
+\end{itemize}
+
+\subsection{Discussione e decisioni}
+\par All'inizio dell'incontro, viene chiesto di esporre al gruppo le attività svolte durante la settimana in vista della fine dello \glossario{sprint}.
+\subsubsection{Analista}
+\par L'Analista ha evidenziato difficoltà nella conversione del documento di \AdR\ e ha stimato a giorni la conclusione della conversione. Sono stati anche approfonditi i casi d'uso relativi al login per comprenderne la fattibilità, evidenziando la possibilità di avere un utente premium in caso di interazioni con l'\glossario{API}. È stata poi espressa la necessità di trovare mezzi e idee per l'approfondimento dei casi d'uso dello strumento di debug fornito al tecnico.
+\par Sono state inoltre evidenziate delle difficoltà sull'ordinamento del documento di \AdR\ date dallo strumento \glossario{LateX}. Infine ha esposto la necessità di un incontro con il Professor. Riccardo Cardin per una funzione di login utente che sarebbe disponibile anche per il tecnico già loggato. Il problema sarebbe arginabile definendo delle precondizioni per il login. 
+\subsubsection{Amministratore}
+\par Vengono discussi i nuovi comandi per la citazione dei documenti in \glossario{LateX}, inseriti nel documento di \NdP. Sono anche avanzati i lavori sul \PdP\ e iniziati i lavori sul \PdQ. Sono anche stati sviluppati tabelle e grafici da inserire per preventivi e consuntivi del \PdP\ e sono iniziati i lavori sui file di configurazione per la compilazione dei file pdf in au1tomatico.
```

Comment:
Si ripetono diversi anche e la parola automatico è scritta scorrettamente 

Sono avanzati i lavori sul \PdP\ e sono iniziati i lavori sul \PdQ. Sono inoltre stati sviluppati tabelle e grafici da inserire per preventivi e consuntivi del \PdP\ e sono iniziati i lavori sui file di configurazione per la compilazione dei file pdf in automatico.

Reply:
Se puoi riesci ad evidenziarmi la sezione che non capisco bene come debba fare la prima correzione?

Reply:
riga 16
```suggestion
Sono avanzati i lavori sul \PdP\ e sono iniziati i lavori sul \PdQ. Sono inoltre stati sviluppati tabelle e grafici da inserire per preventivi e consuntivi del \PdP\ e sono iniziati i lavori sui file di configurazione per la compilazione dei file pdf in automatico.
```